media/css/firefox/whatsnew/whatsnew-67.scss

```diff
@@ -0,0 +1,500 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/call-out';
+@import '../../../protocol/css/components/feature-card';
+@import '../../../protocol/css/components/hero';
+@import '../../../protocol/css/components/modal';
+@import '../../../protocol/css/components/newsletter-form';
+
+@import '../../protocol/components/state-fxa';
+
+
+main {
+    color: $color-off-black;
```

Comment:
Not a blocker, but could this be something we land in Protocol first? 

I know this is what the mockup shows, but I think we need to start being more concious of creating inconsistency and tech-debt across pages. This feels somewhat non-critical, and something that could be introduced with little friction site wide, as opposed to on a page by page basis that we then clean up later?

Reply:
I agree with you. And the cat is already out of the bag on this one, there's several pages with the new colour and it was approved by stakeholders.

Reply:
I’m guessing time is of the essence here and we’re gonna ship with this, but fwiw I think we can change things like this if we decide it’s the right thing to do, even after stakeholders have approved a mock-up. Let’s chalk this small palate cut up and lay that debt later 😉